template/utils/setErrors.ts

```diff
@@ -12,7 +12,7 @@ export function setErrorsFromGraphQLErrors(setError: SetErrorFn, errors: ErrorRe
   });
 }
 
-type SetErrorFn = (e: string, obj: ErrorOption) => void;
+type SetErrorFn = (e: string, obj: typeof ErrorOption) => void;
```

Comment:
@MariahGrey curious as to why we need `typeof` here? Is the `ErrorOption` default export not a typescript object?